misc/output_test.py

```diff
@@ -281,6 +286,70 @@ def test_issue_2048(self) -> None:
             except subprocess.CalledProcessError as err:
                 self.fail("non-zero exit code with: " + err.output)
 
+    def test_pr_2540(self)->None:
+        plan = '''\
+rule CUSTOM_COMMAND
+  command = $COMMAND
+
+build 124: CUSTOM_COMMAND
+  COMMAND = python3 -c 'exit(124)'
```

Comment:
Please do not assume that `python3` is available when running the test (as a general rule, `python3` is never available on Windows, nor on OS X iirc). Instead define a local variable such as `python = sys.executable` and use it in formatted strings for the build plan and the expected results below.